src/main/java/gyro/google/compute/ComputeDistributionPolicy.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.api.services.compute.model.DistributionPolicy;
+import com.google.api.services.compute.model.DistributionPolicyZoneConfiguration;
+import gyro.core.resource.Diffable;
+import gyro.core.validation.Required;
+import gyro.google.Copyable;
+
+public class ComputeDistributionPolicy extends Diffable implements Copyable<DistributionPolicy> {
+
+    private List<ComputeDistributionPolicyZoneConfiguration> zone;
+
+    /**
+     * Zones where the regional managed instance group will create and manage instances.
+     *
+     * @subresource gyro.google.compute.ComputeDistributionPolicyZoneConfiguration
+     */
+    @Required
+    public List<ComputeDistributionPolicyZoneConfiguration> getZone() {
+        if (zone == null) {
+            zone = new ArrayList<>();
+        }
+
+        return zone;
+    }
+
+    public void setZone(List<ComputeDistributionPolicyZoneConfiguration> zone) {
+        this.zone = zone;
+    }
+
+    public DistributionPolicy copyTo() {
+        DistributionPolicy distributionPolicy = new DistributionPolicy();
+        List<ComputeDistributionPolicyZoneConfiguration> zone = getZone();
+
+        if (!zone.isEmpty()) {
+            distributionPolicy.setZones(zone.stream()
+                .map(ComputeDistributionPolicyZoneConfiguration::copyTo)
+                .collect(Collectors.toList()));
+        }
```

Comment:
As zone is a required field, isn't the check redundant ?
we could directly set it
```
 distributionPolicy.setZones(zone.stream()
    .map(ComputeDistributionPolicyZoneConfiguration::copyTo)
    .collect(Collectors.toList()));
```

Reply:
Annotation doesn't guarantee the value is not `null`.